requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 Django>=4.0,<4.1
 django-dotenv==1.4.1
-wagtail>=2.16,<2.17
+wagtail>=2.16,<4
```

Comment:
This should be `wagtail>=3.0,<4` since the new `from wagtail import ...` lines won't work on 2.16.